### DISCUSSION TITLE
Extension demo panel

### DISCUSSION THREAD

**Discussion by pppaw** [foxglove/community#778] [Q&A]
I can't see extension demo panel in my foxglove layout. I have attached layout snapshot of my installed foxglove studio. Can anyone guide me how to open demo extension panel?
![Capture](https://github.com/foxglove/community/assets/112718084/3b2f332c-60f1-43ba-a0f8-1b541bf8d11a)

**Comment by defunctzombie** [comment]
@pppaw You've selected MCAP as the category for your question - is there something mcap specific about your question?

**Reply by pppaw** [reply]
I have to build custom panel. to understand demo custom panel I followed all given steps in foxglove doc. But to reflect result in foxglove there should be extension panel demo panel option in foxglove layout, which I can not see in my foxglove account. I am looking for marked option.
![Capture1](https://github.com/foxglove/community/assets/112718084/dc318ab1-5e5a-4335-9475-a10a54d7e9af)

**Reply by defunctzombie** [reply]
Did you go through any of the steps to install the panel?

**Comment by pppaw** [comment]
Yes, I went through all steps to see demo extension. But to see demo in foxglove extension demo panel is required. which I can't see in my foxglove layout. If you compare both images, both foxglove layouts are different.

**Reply by defunctzombie** [reply]
I'm sorry but I don't understand what you are describing. Could you clarify the steps you took to build, package, and install the extension? Maybe provide a video or set of screenshots showing the process and then how you are not seeing your new extension in the list of panels?

It would also be helpful to see the code for your extension to make sure you've called the appropriate functions to register a new panel. Without such details it is hard to help.
